qlib/cc/sev_snp/ghcb.rs

```diff
@@ -0,0 +1,593 @@
+use core::ptr;
```

Comment:
You may need to add a header here to clarify where you got the code, and who modified the code